hardware_interface/include/hardware_interface/hardware_info.hpp

```diff
@@ -50,17 +50,17 @@ struct HardwareInfo
    * \brief map of joints provided by the hardware where the key is the joint name.
    * Required for Actuator and System Hardware.
    */
-  std::unordered_map<std::string, components::ComponentInfo> joints;
+  std::vector<components::ComponentInfo> joints;
```

Comment:
@destogl are these intentional or remainder of merging master?

Reply:
Yes, it should be a vector type.